b2sdk/transfer/outbound/copy_manager.py

```diff
@@ -149,9 +151,12 @@ def _copy_part(
         return response
 
     def _copy_small_file(
-        self, copy_source, file_name, content_type=None, file_info=None, destination_bucket_id=None
+        self, copy_source, file_name, content_type=None, file_info=None, destination_bucket_id=None,
+        progress_listener=None,
     ):
-        # no progress report - because there is nothing to report
+        if progress_listener is not None:
+            progress_listener.set_total_bytes(copy_source.get_content_length() or 0)
```

Comment:
```suggestion
            progress_listener.set_total_bytes(copy_source.get_content_length() or 1)
```
?
This will make the progress listener at least a little bit useful

Reply:
rejected